src/database/schema.ts

```diff
@@ -0,0 +1,34 @@
+import {modelOptions, prop, PropType} from "@typegoose/typegoose";
+
+@modelOptions({ schemaOptions: { collection: 'room'}})
+export class RoomSchema {
```

Comment:
can you change the naming of the fields for the schema to use CamelCase instead of snake_case.

Reply:
good catch. For the future, let's use [NIT] for small issues like this so we can merge into dev without having to request a follow up.